sdk/node/Libplanet.Node/Services/BlockChainService.cs

```diff
@@ -152,4 +174,83 @@ private static Block LoadGenesisBlockFromUrl(Uri genesisBlockUri)
         var blockDict = (Dictionary)_codec.Decode(rawBlock);
         return BlockMarshaler.UnmarshalBlock(blockDict);
     }
+
+    private static byte[] LoadConfigurationFromFilePath(string configurationPath)
+    {
+        return File.ReadAllBytes(Path.GetFullPath(configurationPath));
+    }
+
+    private static byte[] LoadConfigurationFromUri(Uri configurationUri)
+    {
+        using var client = new HttpClient();
+        return configurationUri.IsFile
+            ? LoadConfigurationFromFilePath(configurationUri.AbsolutePath)
+            : client.GetByteArrayAsync(configurationUri).Result;
+    }
+
+    private static Block CreateGenesisBlockFromConfiguration(
```

Comment:
How about throwing an exception instead of handling the error or performing alternative actions if the data for creating the genesis block is invalid?

```csharp
        if (data == null || data.Count == 0)
        {
            throw new .....
        }
```
```csharp
                    catch (Exception e)
                    {
                        // skip state
                        throw e;
                    }
```

Reply:
Throwing an error in the service, the exception bubble pops up to the program?

Reply:
Genesis is the beginning of blockchain, so shouldn't it be perfect? It seems to me that if the initialization is wrong, it's like not knowing which direction the process will go, so if it's wrong, the blockchain shouldn't start at all.

Reply:
IMO this would be better to be in `planet` CLI :thinking: